Makefile

```diff
@@ -18,3 +21,7 @@ e2e: build
 .PHONY: build
 build:
```

Comment:
Lets run generate for every build to keep it sync with the code

```suggestion
build: generate
```

Reply:
fixed